packages/admin/cms-admin/src/userPermissions/user/permissions/ContentScopeGridToolbar.tsx

```diff
@@ -0,0 +1,58 @@
+import { Select } from "@comet/admin-icons";
+import { Button, styled } from "@mui/material";
+import { GridToolbarContainer, GridToolbarFilterButton, GridToolbarQuickFilter } from "@mui/x-data-grid";
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+type Props = {
+    actions?: { deleteSelected: () => void; openDialog: () => void };
```

Comment:
This props should be required

Reply:
but in the case of the dialog these actions should not be available